kaiax/gov/headergov/impl/init.go

```diff
@@ -154,8 +155,12 @@ func (h *headerGovModule) scanAllVotesInEpoch(epochIdx uint64) map[uint64]header
 			logger.Error("Failed to parse vote", "num", blockNum, "err", err)
 			continue
 		}
-		// TODO-kaiax: consider writing addval/removeval votes to validator DB.
-		if vote != nil && vote.Name() != "governance.addvalidator" && vote.Name() != "governance.removevalidator" {
+
+		if vote == nil {
```

Comment:
Will addval/removeval vote blocks be stored in VoteDataBlockNum?

Reply:
No. This check is moved to below lines.

Reply:
Done 613ad27b